src/neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -219,7 +219,12 @@ private StackItem GetValidators(ApplicationEngine engine, Array args)
         public ECPoint[] GetValidators(StoreView snapshot)
         {
             StorageItem storage_count = snapshot.Storages.TryGet(CreateStorageKey(Prefix_ValidatorsCount));
-            if (storage_count is null) return Blockchain.StandbyValidators;
+            if (storage_count is null)
+            {
+                ECPoint[] validators = new ECPoint[Blockchain.StandbyValidators.Length];
+                System.Array.Copy(Blockchain.StandbyValidators, validators, validators.Length);
```

Comment:
It's not safe here, I've added a temporary array here to prevent the data from being tampered outside.

Reply:
Could you create another PR in master, because oracle will spend more time in be merged

Reply:
Okay, PR #1458. I'll reset it, but ut will fail.